### DISCUSSION TITLE
what commit of llama.cpp does v0.1.57 of llama-cpp-python use?

### DISCUSSION THREAD

**Discussion by siddhsql** [abetlen/llama-cpp-python#338] [Q&A] [answered]
Hello, I am wondering what commit of llama.cpp is used by v0.1.57 of llama-cpp-python. could anyone tell me please? thanks.

**Comment by gjmulder** [comment, marked as answer]
Moved to Q&A as this isn't an issue. You could simply browse to the tag for 0.1.57 and look:

https://github.com/abetlen/llama-cpp-python/tree/v0.1.57/vendor

**Reply by s7726** [reply]
I'm using the GitHub mobile app and can't find a way to figure out what commit or version is pulled in for all the "update llama.cpp" commits.

Would be great if the commit message included more info.

**Comment by siddhsql** [comment]
Thanks a lot. when I cloned the repo I just got an empty vendor directory on my computer but on github.com I see its linked to the commit. How is this possible?

**Reply by gjmulder** [reply]
Did you use: `git clone --recurse-submodules` ?

**Comment by siddhsql** [comment]
nope. first time i am seeing that command.

**Reply by gjmulder** [reply]
https://github.com/abetlen/llama-cpp-python#development
